### PR TITLE
TNT-40406 - use crypto random when available

### PR DESCRIPTION
**Pull request by artur-ciocanu**
Ensure UUID is generated using crypto random instead of `Math.random()`

## Description

Ensure UUID is generated using crypto random instead of `Math.random()`

## Related Issue

None

## Motivation and Context

Ensure UUID is generated using crypto random instead of `Math.random()`

## How Has This Been Tested?

Unit tests

## Screenshots (if appropriate):
None

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by 5amfung — APPROVED** [review]
LGTM

File: packages/target-tools/src/uuid/stringify.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const byteToHex = [];
+
+for (let i = 0; i < 256; i += 1) {
```
Comment (XDex) [inline packages/target-tools/src/uuid/stringify.js:3]:
```suggestion
for (let i = 0; i < 256; i++) {
```

File: packages/target-tools/src/uuid/stringify.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    byteToHex[arr[offset + 6]] +
+    byteToHex[arr[offset + 7]] +
+    byteToHex[arr[offset + 8]] +
+    byteToHex[arr[offset + 9]] +
+    byteToHex[arr[offset + 10]] +
+    byteToHex[arr[offset + 11]] +
+    byteToHex[arr[offset + 12]] +
+    byteToHex[arr[offset + 13]] +
+    byteToHex[arr[offset + 14]] +
+    byteToHex[arr[offset + 15]]
```
Comment (XDex) [inline packages/target-tools/src/uuid/stringify.js:25]: can we use a for loop for this?

**Review by XDex — APPROVED** [review]
@artur-ciocanu Please take a look at some suggested fixes, otherwise looks good

File: packages/target-tools/src/uuid/stringify.js (hunk 1 above)
Comment (artur-ciocanu) [reply packages/target-tools/src/uuid/stringify.js:3]: Lint error.

File: packages/target-tools/src/uuid/stringify.js (hunk 2 above)
Comment (artur-ciocanu) [reply packages/target-tools/src/uuid/stringify.js]: Done.

**Pull request merged**